prisma-fmt/src/lib.rs

```diff
@@ -65,6 +66,28 @@ pub fn lint(schema: String) -> String {
     lint::run(&schema)
 }
 
+/// Function that emulates what happened when improperly calling the `getDmmf` query engine RPC on invalid schemas.
```

Comment:
Historical comments are useful when reviewing the PR, but they tend to get confusing very fast for a reader that comes to the docs through other paths.